Sources/Sandbox.Game/Game/Entities/Inventory/MyInventory.cs

```diff
@@ -193,7 +193,7 @@ public bool CanItemsBeAdded(MyFixedPoint amount, MyDefinitionId contentId)
             var adapter = MyInventoryItemAdapter.Static;
             adapter.Adapt(contentId);
 
-            if (MyPerGameSettings.ConstrainInventory() && (amount * adapter.Volume + m_currentVolume > MaxVolume) || (amount * adapter.Mass + m_currentMass > m_maxMass))
+            if (MyPerGameSettings.ConstrainInventory() && (amount * adapter.Volume + m_currentVolume >= MaxVolume) || (amount * adapter.Mass + m_currentMass > m_maxMass))
```

Comment:
yes, atm. inventory has only volume limit, but it might change in feature. so that check might be worth adding.